{
  "uid" : "c8d8a3e79caa610c",
  "name" : "Auto-generation with different configurations -- @1.2 ",
  "fullName" : "Auto-Generated Issuing CAs: Auto-generation with different configurations",
  "historyId" : "7779bcba3d5734bc5d8a5305ff78ca73",
  "time" : {
    "start" : 1746711073995,
    "stop" : 1746711074565,
    "duration" : 570
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: When the admin sets the following CA configuration\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 49, in step_when_admin_sets_ca_config\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: When the admin sets the following CA configuration\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 49, in step_when_admin_sets_ca_config\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746711073997,
        "stop" : 1746711074536,
        "duration" : 539
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the admin sets the following CA configuration:",
      "time" : {
        "start" : 1746711074537,
        "stop" : 1746711074537,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: When the admin sets the following CA configuration\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 49, in step_when_admin_sets_ca_config\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "e32d11d8e768cd51",
        "name" : ".table",
        "source" : "e32d11d8e768cd51.csv",
        "type" : "text/csv",
        "size" : 59
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then the system generates a CA with:",
      "time" : {
        "start" : 1746711074565,
        "stop" : 1746711074565,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "9408fd1fc51022fa",
        "name" : ".table",
        "source" : "9408fd1fc51022fa.csv",
        "type" : "text/csv",
        "size" : 59
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Auto-Generated Issuing CAs"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "key_size",
    "value" : "4096"
  }, {
    "name" : "subject_name",
    "value" : "Secure_CA"
  }, {
    "name" : "validity",
    "value" : "730"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "b05591d9d38d423e",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/21/#testresult/b05591d9d38d423e",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin sets the following CA configuration\n",
        "time" : {
          "start" : 1746710239764,
          "stop" : 1746710240334,
          "duration" : 570
        }
      }, {
        "uid" : "38192143cd27cba8",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/38192143cd27cba8",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin sets the following CA configuration\n",
        "time" : {
          "start" : 1746709785893,
          "stop" : 1746709786463,
          "duration" : 570
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "c8d8a3e79caa610c.json",
  "parameterValues" : [ "4096", "Secure_CA", "730" ]
}